node-feature-discovery-daemonset.json.template

```diff
@@ -0,0 +1,43 @@
+{
+  "apiVersion": "apps/v1",
+  "kind": "DaemonSet",
+  "metadata": {
+    "labels": {
+      "app": "node-feature-discovery"
+    },
+    "name": "node-feature-discovery"
+  },
+  "spec": {
+    "selector": {
+      "matchLabels": {
+        "app": "node-feature-discovery"
+      }
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "app": "node-feature-discovery"
+        }
+      },
+      "spec": {
+        "hostNetwork": true,
+        "containers": [
+          {
+            "env": [
+              {
+                "name": "NODE_NAME",
+                "valueFrom": {
+                  "fieldRef": {
+                    "fieldPath": "spec.nodeName"
+                  }
+                }
+              }
+            ],
+            "image": "quay.io/kubernetes_incubator/node-feature-discovery:v0.1.0",
+            "name": "node-feature-discovery"
```

Comment:
May be we should add the `--sleep-interval` arg in this template to show it as an example? This can also be linked in the doc?